Mathlib/NumberTheory/LucasPrimality.lean

```diff
@@ -59,3 +58,74 @@ theorem lucas_primality (p : ℕ) (a : ZMod p) (ha : a ^ (p - 1) = 1)
     p - 1 = orderOf a := order_of_a.symm
     _ = orderOf a' := (orderOf_injective (Units.coeHom (ZMod p)) Units.ext a')
     _ ≤ Fintype.card (ZMod p)ˣ := orderOf_le_card_univ
+
+
+/-- If `p` is prime, then there exists an `a`
```

Comment:
There's a lot of awkward space here, can you reindent the line to fully utilize the 100 characters? (or at least, split it at some point that makes more sense).